static/app/views/starfish/utils/useSpansQuery.tsx

```diff
@@ -55,6 +71,35 @@ function useWrappedQuery(queryString: string, initialData?: any) {
   return {isLoading, data};
 }
 
+function useWrappedDiscoverTimeseriesQuery(eventView: EventView, initialData?: any) {
+  const location = useLocation();
+  const organization = useOrganization();
+  const {isLoading, data} = useGenericDiscoverQuery<
+    {
+      data: any[];
+    },
+    DiscoverQueryProps
+  >({
+    route: 'events-stats',
+    eventView,
+    location,
+    orgSlug: organization.slug,
+    getRequestPayload: () => ({
+      ...eventView.getEventsAPIPayload(location),
+      yAxis: Array.from(
+        new Set([eventView.yAxis, ...eventView.fields.map(f => f.field)])
+      ),
```

Comment:
Can't you use the queryExtras prop to set the yAxis?

Reply:
Yeah queryExtras also works but the type is `queryExtras?: Record<string, string>;` so it complains if I pass an array value for yAxis (even though it works). I figured I'd just do it this way instead of altering or forcing the type.

Reply:
Personally i'd be against forcing a type, but updating the type seems like the best option IMO.
I've tested queryExtras with a string array and it works perfectly, I think the type of queryExtras should reflect that.